BGL/include/CGAL/boost/graph/Face_filtered_graph.h

```diff
@@ -71,9 +72,9 @@ namespace CGAL
    * \cgalModels \bgllink{VertexListGraph}
    */
 template<typename Graph,
-         typename FIMap = typename boost::property_map<Graph, CGAL::face_index_t>::type,
-         typename VIMap = typename boost::property_map<Graph, boost::vertex_index_t>::type,
-         typename HIMap = typename boost::property_map<Graph, CGAL::halfedge_index_t>::type>
+         typename FIMap = typename CGAL::GetInitializedFaceIndexMap<Graph>::const_type,
+         typename VIMap = typename CGAL::GetInitializedVertexIndexMap<Graph>::const_type,
```

Comment:
Get_initialized_vertex_index_map?

Reply:
Every other getter in named_params_helper is using caml case for some reason. This can be changed but should be done CGAL-wide in a different PR, I'd say.